src/generator/generator.ts

```diff
@@ -1,7 +1,10 @@
-import {FunctionExpression, isIdentifier, LVal,} from '@babel/types';
+import {FunctionExpression} from '@babel/types';
 import {i32, Module} from 'binaryen';
+import DeclarationVisitor from './declaration_visitor';
 import GeneratorVisitor from './generator_visitor';
 
+type Mapping = Map<string, number>;
```

Comment:
We should move this type definition to the visitor base class file, to prevent a coupling of the visitors to the generator.